README.md

```diff
@@ -252,20 +252,24 @@ The following generic methods are added to `nemo.view`
 
 `@returns {Promise}` resolves to true or rejected
 
-#### _wait(locatorString[, timeout])
+#### _wait(locatorString[, timeout, msg])
```

Comment:
May seem like a trivial point, but the syntax above suggests that timeout and msg are both required together. Suggest:

`_wait(locatorString[, timeout[, msg]])